Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG DOCKER_REGISTRY
 ARG ALPINE_VERSION=3.15
```

Comment:
I would change alpine version here also to the latest one 3.17

Reply:
Done